### DISCUSSION TITLE
What's the MyST syntax for default_role?

### DISCUSSION THREAD

**Discussion by arwedus** [executablebooks/meta#940] [Q&A]
Hi @chrisjsewell and all, I wondered whether I can make use of the Sphinx "default_role" as comfortable as in rst, where you just have to write like this:

```
The function is implemented in `myFunction()`
```

and if you have set `default_role` to the `cpp` role or the `any` role, you can really save time by being able to omit the role almost everywhere.

So, what would be the syntax for MyST? Since Markdown already has defined single backticks as inline verbatim text, would it be:

```
{}`myFunction()`
```

Or could we just overload the single backticks to be either the default_role (if it resolves something) or verbatim text otherwise  (probably not)?

**Comment by drothlis** [comment]
Related: https://github.com/executablebooks/MyST-Parser/discussions/648
